diff --git a/contracts/rewards/RewardsDistributor.sol b/contracts/rewards/RewardsDistributor.sol
index 04d057a..93b5f78 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -306,7 +306,8 @@ abstract contract RewardsDistributor is IRewardsDistributor {
       rewardData.index = uint104(newIndex);
       rewardData.lastUpdateTimestamp = block.timestamp.toUint32();
     } else {
-      rewardData.lastUpdateTimestamp = block.timestamp.toUint32();
+      // Mutate lastUpdateTimestamp and have it one sec in the future - should break an invariant
+      rewardData.lastUpdateTimestamp = block.timestamp.toUint32() + 1;
     }
 
     return (newIndex, indexUpdated);
